usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -3746,9 +3755,13 @@ CK_RV token_specific_object_add(STDLL_TokData_t * tokdata, SESSION * sess,
     CK_ATTRIBUTE *attr = NULL;
     CK_BYTE blob[MAX_BLOBSIZE];
     size_t blobsize = sizeof(blob);
+    CK_BYTE spki[MAX_BLOBSIZE];
+    CK_ULONG spkisize = sizeof(spki);
     CK_RV rc;
```

Comment:
From review together with Vishnu: here you are using CK_ULONG spkisize, but in the functions import_xxxxx_key() you are using size_t *spki_size. Yes, it's the same, but just to mention. For comparison: blob_size is a size_t at both places.

Reply:
Ah right. Will change it to size_t here as well (for consistency)